scripts/build-windows-k8s.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+set -eo pipefail
+
+ACS_ENGINE_HOME=${GOPATH}/src/github.com/Azure/acs-engine
+
+usage() {
+	echo "$0 [-v version] [-p acs_patch_version]"
+	echo " -v <version>: version"
+	echo " -p <patched version>: acs_patch_version"
+	exit 0
+}
+
+while getopts ":v:p:" opt; do
+  case ${opt} in
+    v)
+      version=${OPTARG}
+      ;;
+    p)
+      acs_patch_version=${OPTARG}
+      ;;
+    *)
+			usage
+      ;;
+  esac
+done
+
+if [ -z "${version}" ] || [ -z "${acs_patch_version}" ]; then
+    usage
+fi
+
+if [ -z "${AZURE_STORAGE_CONNECTION_STRING}" ] || [ -z "${AZURE_STORAGE_CONTAINER_NAME}" ]; then
+    echo '$AZURE_STORAGE_CONNECTION_STRING and $AZURE_STORAGE_CONTAINER_NAME need to be set for upload to Azure Blob Storage.'
+		exit 1
+fi
+
+KUBERNETES_RELEASE=$(echo $version | cut -d'.' -f1,2)
+KUBERNETES_RELEASE_BRANCH_NAME=release-${KUBERNETES_RELEASE}
+ACS_VERSION=${version}-${acs_patch_version}
+ACS_BRANCH_NAME=acs-v${ACS_VERSION}
+DIST_DIR=${ACS_ENGINE_HOME}/_dist/k8s-windows-v${ACS_VERSION}
+
+fetch_k8s() {
+	git clone https://github.com/Azure/kubernetes ${GOPATH}/src/k8s.io/kubernetes || true
+	cd ${GOPATH}/src/k8s.io/kubernetes
+	git remote add upstream https://github.com/kubernetes/kubernetes || true
+	git fetch upstream
+}
+
+set_git_config() {
+	git config user.name "ACS CI"
+	git config user.email "containers@microsoft.com"
+}
+
+create_version_branch() {
+	git checkout -b ${ACS_BRANCH_NAME} ${KUBERNETES_RELEASE_BRANCH_NAME} || true
+}
+
+create_dist_dir() {
+	mkdir -p ${DIST_DIR}
+}
+
+build_kubelet() {
+	echo "building kubelet.exe..."
+	build/run.sh make WHAT=cmd/kubelet KUBE_BUILD_PLATFORMS=windows/amd64
+	cp ${GOPATH}/src/k8s.io/kubernetes/_output/dockerized/bin/windows/amd64/kubelet.exe ${DIST_DIR}
```

Comment:
What's "dockerized" directory? It used to be "local"?

Reply:
That's the path when using docker to build Kubernetes as opposed to native (golang on system).

Reply:
Yep!

Reply:
In that case, can you add a comment or in md so that people don't run it natively?

Reply:
I'll add some more information about the Kubernetes build process and include this. Good suggestion!

Reply:
Called out Docker and added more background in the doc.